test/models/report_test.rb

```diff
@@ -3,7 +3,32 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+
+  test '#editable?' do
+    alice_report = reports(:alice_report)
+
+    assert alice_report.editable?(users(:alice))
+    assert_not alice_report.editable?(users(:bob))
+  end
+
+  test '#created_on' do
+    report = reports(:alice_report)
+    assert_equal report.created_at.to_date, report.created_on
```

Comment:
```suggestion
    report = reports(:alice_report, created_at: "2024-02-25 16:00".in_time_zone)
    assert_equal "2024-02-25".to_date, report.created_on
```

参考：明示的にデータを記述することでテストコードの可読性が上がります。（パッと見で仕様が理解しやすくなる）

https://speakerdeck.com/jnchito/number-vstat?slide=34

Reply:
修正しました! ご提案いただいた形そのままですと以下エラーが発生してしまったため、reportの用意の仕方を少し変えました。
Minitest::UnexpectedError: StandardError: No fixture named '{:created_at=>Tue, 27 Feb 2024 21:00:00.000000000 JST +09:00}' found for fixture set 'reports'